src/components/scene/wakelock.js

```diff
@@ -0,0 +1,15 @@
+var register = require('../../core/component').registerComponent;
+var Wakelock = require('../../../vendor/wakelock/wakelock');
```

Comment:
when you attach a component to an entity it becomes part of its public API. Things that we don't want to expose to the outside world should stay as internal modules instead of being a component.